examples/examples-cpu/prefect/README.md

```diff
@@ -1,3 +1,5 @@
 # prefect-scheduled-scoring
 
-The files in this directory were created to support ["Using a scheduled prefect flow with a Saturn Dask cluster"](http://docs.saturncloud.io/en/articles/4135965-using-a-scheduled-prefect-flow-with-a-saturn-dask-cluster)
+The files in this directory were created to support these articles:
+
+  * ["Scheduled Data Pipelines"](https://www.saturncloud.io/docs/deployments/data-pipelines)
```

Comment:
I know this seems a little weird when there's only one link, but I'm working on a Prefect Cloud addition right now that will be ready soon.